src/xdp.props

```diff
@@ -28,4 +18,21 @@
       <Platform>ARM64</Platform>
     </ProjectConfiguration>
   </ItemGroup>
+  <PropertyGroup>
+    <!-- Defines the XDP version numbers -->
+    <XdpMajorVersion>1</XdpMajorVersion>
+    <XdpMinorVersion>1</XdpMinorVersion>
+    <XdpPatchVersion>0</XdpPatchVersion>
+    <!-- Project-wide properties -->
+    <EbpfPackagePath>$(SolutionDir)packages\eBPF-for-Windows.0.17.0\</EbpfPackagePath>
+    <WiXPackagePath>$(SolutionDir)packages\wix.3.14.1\</WiXPackagePath>
+    <WntPackagePath>$(SolutionDir)packages\win-net-test.0.5.2\</WntPackagePath>
+    <!-- Configuration properties to match Windows -->
+    <WinConfig Condition="'$(Configuration)' == 'Release'">fre</WinConfig>
```

Comment:
is this because all the scripts invoke the build as debug/release instead of fre/chk, and so on?

Reply:
Yes. Those are unchanged.